scripts/start_dev

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
 cd $(dirname $0)/..
-python3 scripts/create_development_docker_compose.py
+if [ ! -f docker-compose.dev.yaml ]; then
```

Comment:
let's run it every time so that the contents are always consistent with the core docker-compose (as scripts/create_development_docker_compose.py takes core docker compose and creates dev one)